### ISSUE TITLE
jx boot failed

### ISSUE THREAD

**Issue by nodece** [jenkins-x/jx#7448]
### Summary 

I tried to boot  jx to cluster failed by `jx boot --requirements jx-requirements.yml`, I got:

```
WARNING: It seems you're passing a requirements file to cluster which has already been provisioned.
WARNING: We recommend you update the jx-requirements.yml file at {  }, using the updates provided within your local jx-requirements.yml file.
error: loading from provided requirements file: attempting to boot cluster using a requirements file which is possibly stale
```

I track jx code found that [devEnv](https://github.com/jenkins-x/jx/commit/b785a5973481b18e105cda3f11acd3d93a42a1e9#diff-2436c7804fbf5ab094833f5aefcbd706R507) was always `non-nil`,  so it failed.

### Steps to reproduce the behavior

`jx boot --requirements jx-requirements.yml`

### Expected behavior

We can boot jx to cluster by `jx boot --requirements jx-requirements.yml`.

### Actual behavior

got the error:

```
WARNING: It seems you're passing a requirements file to cluster which has already been provisioned.
WARNING: We recommend you update the jx-requirements.yml file at {  }, using the updates provided within your local jx-requirements.yml file.
error: loading from provided requirements file: attempting to boot cluster using a requirements file which is possibly stale
```

### Jx version

The output of `jx version` is:

```
Version        2.1.99
Commit         9ac01b9
Build date     2020-07-14T07:24:37Z
Go version     1.13.8
Git tree state clean
```

**Comment by nodece** [comment]
PR: #7431

**Comment by nodece** [comment]
fixed by #7431
